newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -342,6 +349,17 @@ export const HomePage = React.memo<Props>(
         [isActive, authenticated, onCloudProjectsChanged]
       );
 
+      // Refresh games list (as one could have been modified using the game dashboard
+      // in the project manager) when navigating to the "Manage" tab.
+      React.useEffect(
+        () => {
+          if (isActive && activeTab === 'manage' && authenticated) {
```

Comment:
This is an unrelated fix (to keep the manage tab and game dashboard from the project manager in sync)